### PR TITLE
TEST - CSIPSTR3 test case and package.

### LINKED ISSUE #21: CSIPSTR3 Test Case Description
**Specification:**
- **Name:** E-ARK CSIP
  **Version:** 2.0-DRAFT
  **URL:** http://earkcsip.dilcis.eu/

**Requirement:**
- **Id:** CSIPSTR3
- **Link:** http://earkcsip.dilcis.eu/#CSIPSTR3

**Corpus Folder:** `corpora/csip/structure/folders/`

**Error Level:** INFO

**Description:**
> The Information Package folder CAN be compressed (for example by using TAR or ZIP);

The requirement should be reworded as TAR is not a compressed format, just an archive format. 

First thoughts here are do we need to go here at all? Archiving / compression could be considered as peripheral to our concerns, we don’t care how you compress and transfer we just care about the uncompressed, non-archived form?

If we want to test this then it doesn’t require specific test cases. Instead we produce archive versions of all of our test packages and present them to the validator. We test two aspects of the validator:
- whether it can recognise and process the archive as an IP
- whether it gathers & reports the correct details, gives appropriate results, etc.

### PR DESCRIPTION
**Pull request by carlwilson**
Factored out from original work by @ltaimre

closes #21

### CODE REVIEW

**Comment by carlwilson** [comment]
Hi @ltaimre I'm proposing a different approach here. The requirement states that IPs can be compressed. We're creating uncompressed/unarchived directory structures. This is a deliberate choice to avoid storing packaged IPs on GitHub, while giving us the option of producing other archive formats for testing. I propose a more general test, rather than produce a specific zipped test package. What I have in mind is to present the full test set in the archive formats we intend to support as well as in directory form and compare the results which should be identical. The compressed/archived versions will be produced by an automated task. This is almost a "see also" task, we need to document the approach in the test case XML is all.

**Review by ltaimre — DISMISSED** [review]

File: corpus/structure/CSIPSTR2/testCase.xsd [hunk 1]
```diff
@@ -0,0 +1,1746 @@
+
```
Comment (carlwilson) [inline corpus/structure/CSIPSTR2/testCase.xsd]: This should be the test case XSD not the HTML page from GitHub

File: corpus/structure/CSIPSTR1/testCase.xsd [hunk 2]
```diff
@@ -0,0 +1,1746 @@
+
```
Comment (carlwilson) [inline corpus/structure/CSIPSTR1/testCase.xsd]: Needs to be the XSD document NOT the HTML page

File: corpus/structure/CSIPSTR3/testCase.xsd [hunk 3]
```diff
@@ -0,0 +1,1746 @@
+
```
Comment (carlwilson) [inline corpus/structure/CSIPSTR3/testCase.xsd]: Again change to XSD

File: corpus/structure/CSIPSTR1/testCase.xsd (hunk 2 above)
Comment (ltaimre) [reply corpus/structure/CSIPSTR1/testCase.xsd]: Done

File: corpus/structure/CSIPSTR2/testCase.xsd (hunk 1 above)
Comment (ltaimre) [reply corpus/structure/CSIPSTR2/testCase.xsd]: Done

**Review by ltaimre — APPROVED** [review]

**Pull request merged**